app/assets/stylesheets/application.scss

```diff
@@ -2,13 +2,92 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
+body {
+  background: #F9F9F9
+}
+
+#face-img {
+  position: relative;
+}
+
+#face-img img{
```

Comment:
Avoid using id selectors<br>Merge rule `#face-img img` with rule on line 9<br>Opening curly brace `{` should be preceded by one space